data/wanderer/wanderers start.txt

```diff
@@ -39,6 +39,7 @@ mission "First Contact: Wanderer (Before Hai)"
 		not "First Contact: Hai: offered"
 		not "First Contact: Unfettered: offered"
 	on offer
+		set "global: First Contact: Wanderer"
```

Comment:
Should we really consider this one enough to set the global condition? Technically the mission name says first contact, but the "true" first contact mission is the one when you know the Hai.

Reply:
Maybe the global version can also be split into Pre-Hai and Post-Hai.

Reply:
The point of this is literally that: The players have met the Wanderers. It's not specifying that they can have conversations with them, or know their backstory, or anything else. Just the strict bare-minimum "know they exist because they have quite literally landed in one of their cities." Of course if people want more complexity, they can add those too. But for a baseline for the purposes of ensuring the player isn't "spoiled" by the reveal that the Wanderers exist, this ensures that.